File: python-client/scripts/create_pod.py
from kubernetes import client, config

config.load_incluster_config()
v1 = client.CoreV1Api()

pod=client.V1Pod()
pod.metadata=client.V1ObjectMeta(name="mypod")
container=client.V1Container(name="nginx")
container.image="nginx:1.7.9"
spec=client.V1PodSpec(containers=[container])
pod.spec = spec
v1.create_namespaced_pod(namespace="default",body=pod)

File: node-client/scripts/create_pod.js
const Api = require('kubernetes-client');

//Get credentials (token and certificate) and api endpoints
const api = new Api.Api(Api.config.getInCluster());

const podJSON = {
    "apiVersion": "v1",
    "kind": "Pod",
    "metadata": {
      "name": "mynginxpod"
    },
    "spec": {
      "containers": [
        {
          "name": "nginx",
          "image": "nginx:1.7.9",
          "ports": [
            {
              "containerPort": 80
            }
          ]
        }
      ]
    }
};

// CREATE POD
api.group('v1').ns('default').pods.post({ body: podJSON });




